packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -73,6 +75,45 @@ const configureSpotlightContainer = ({'data-spotlight-id': spotlightId, focusabl
 	});
 };
 
+/*
+ * Track the last position of the pointer to check if a list should scroll by
+ * page up/down keys when the pointer is on a list without any focused node.
+ * `keydown` event does not occur if there is no focus on the node and
+ * its descendants, we add `keydown` handler to `document` also.
+ */
+const
+	lastPointer = {x: 0, y: 0},
+	pointerTracker = (ev) => {
+		lastPointer.x = ev.clientX;
+		lastPointer.y = ev.clientY;
+	};
+
+const
+	// An app could have lists and/or scrollers more than one,
+	// so we should test all of them when page up/down key is pressed.
+	scrollables = new Set(),
+	pageKeyHandler = (ev) => {
+		if (Spotlight.getPointerMode() && !Spotlight.getCurrent()) {
+			const
+				{x, y} = lastPointer,
+				elem = document.elementFromPoint(x, y);
+
+			if (elem) {
+				for (let s of scrollables) {
+					if (ReactDOM.findDOMNode(s).contains(elem)) { // eslint-disable-line react/no-find-dom-node
```

Comment:
It'd be nice to avoid this `findDOMNode` if possible. Since we already have various node references in Scrollable, can we provide it to this construct instead. For example, instead of passing the React internal instance, we could use a WeakMap to map the node reference to the handler.

Something (loosely) like:

```
scrollables = new WeakMap();
pageKeyHandler = (ev) => {
  // existing code ...
  for (let node of scrollables.keys()) {
    if (node.contains(elem)) {
      scrollables.get(node)(ev);
      break;
    }
 }

// in the component
scrollables.set(this.someRef.current, this.onGlobalKeyDown);
```

Reply:
Unfortunately `WeakMap` keys are not enumerable, so I can use `Set` or `Map`. And it looks not good to use _node_ as a key if we don't use `WeakMap`. So I kept _this_ as a key on `Map`.